frontends/web/src/api/exchanges.ts

```diff
@@ -101,3 +101,13 @@ export const getExchangeBuySupported = (code: string) => {
     return apiGet(`exchange/buy-supported/${code}`);
   };
 };
+
+export const getExchangeSupportedAccounts = async (accounts: IAccount[]): Promise<IAccount[]> => {
+  const accountsWithFalsyValue = await Promise.all(
+    accounts.map(async (account) => {
+      const supported = await getExchangeBuySupported(account.code)();
+      return supported.exchanges.length ? account : false;
+    })
+  );
+  return accountsWithFalsyValue.filter(result => result) as IAccount[];
+};
```

Comment:
nit: all other functions in api just add types to get/post responses. This one is more like a helper function, would it make sense to have a backend endpoint that takes ` IAccount[]` ?

alternatives move to `src/routes/buy/utils` or similar?